### DISCUSSION TITLE
FontAwesome 패키지 제거 건

### DISCUSSION THREAD

**Discussion by healtheloper** [Co-Studo/Co-Studo-front#97] [Polls]
@jindonyy @hemudi @mina-gwak 

<img width="1244" alt="스크린샷 2022-10-27 오후 8 34 24" src="https://user-images.githubusercontent.com/58503584/203701247-2177ac5d-53f5-4139-96cd-8d7018704b36.png">

FontAwesome 은 dependency 중에서 5번째로 큰 패키지 입니다

장점은 Github Icon 외에도 디자인에 존재하지 않는 아이콘을 가져오기 쉽습니다
단점은 번들사이즈가 커진다는 점입니다

저는 asset 기반 Icon 을 import 하는 방식도 사용하고 있으므로 장점대비 단점이 크다고 생각하여 제거하는게 좋다고 생각합니다

**Comment by hemudi** [comment]
우와 엄청 컸네요...
말씀해주신대로 이미 Icon을 관리하고 사용하는 다른 방식을 적용한 상태이니
유지보수를 위해 Icon 사용 방식 통일하는 차원에서도 제거하는게 좋을 것 같습니다!
